.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "hardhat/coordinape-protocol"]
-	path = hardhat/coordinape-protocol
+[submodule "hardhat/contracts/coordinape-protocol"]
```

Comment:
I have done this to make it easy for us to add more contracts in future. We will need to add some yearn contracts for testing migration and it would be nice to add it as it's own submodule instead of including it into coordinape-protocol. Let me know if you think otherwise

Reply:
sounds good!